include/zephyr/sys/util.h

```diff
@@ -259,6 +259,24 @@ extern "C" {
  */
 #define DIV_ROUND_UP(n, d) (((n) + (d) - 1) / (d))
 
+#ifndef DIV_ROUND_NEAREST
```

Comment:
We had few conversations about guards for macros and I think it was decided that we are indeed the center of the known (software) universe and we won't add these, but rather change modules and downstream applications instead.

@carlescufi @nashif should these guards be dropped?